### DISCUSSION TITLE
Seeing an error when authenticating with GitHub

### DISCUSSION THREAD

**Discussion by jrschumacher** [trunk-io/docs#270] [Q&A] [answered]
![CleanShot 2024-06-01 at 12 14 54@2x](https://github.com/trunk-io/docs/assets/46549/e50e8396-a8d3-4810-9f5a-c9ac561555a7)

**Comment by joshmarinacci** [comment]
Hi. I'm sorry this happened to you.  I've informed the dev team and they are looking into it. This definitely shouldn't be happening.

**Reply by jrschumacher** [reply]
Is there a better place to report issues? This was the only place I could find to submit general issues.

**Reply by gewenyu99** [reply]
Hey, this is the right place to submit issues. You can always find us on [Slack](https://slack.trunk.io/) as well for a shorter response time. If you'd like to debug something with us or need to DM us with more information, join us there.

Cheers.

**Reply by puzzler7** [reply]
Hi, there's a bug at the `/getting-started` URL that we're working on resolving now. In the meantime, you should be able to visit [app.trunk.io/onboarding](https://app.trunk.io/onboarding) to get unblocked.

**Reply by puzzler7** [reply, marked as answer]
This should be fixed now. Let us know if you run into any more problems!
